presto-product-tests/bin/run_on_docker.sh

```diff
@@ -208,6 +208,8 @@ trap terminate INT TERM EXIT
 
 if [[ "$ENVIRONMENT" == "singlenode-sqlserver" ]]; then
   EXTERNAL_SERVICES="hadoop-master sqlserver"
+elif [[ "$ENVIRONMENT" == "singlenode-ldap" ]]; then
+  EXTERNAL_SERVICES="hadoop-master ldapserver cassandra"
```

Comment:
Why do we need `cassandra` here even though we don't have it in `singlenode-sqlserver` `EXTERNAL_SERVICES`?
Do we have any ldap tests with `cassandra`?